client/src/components/TeamPage/CardTest.js

```diff
@@ -0,0 +1,225 @@
+import React from "react";
+import {
+  AiFillFacebook,
+  AiFillTwitterCircle,
+  AiFillLinkedin,
+  AiFillYoutube,
+} from "react-icons/ai";
+import "./CardTest.css";
+const CardTest = () => {
+  return (
+    <>
+      <h2 class="design">Design Team</h2>
+      <div class="box">
+        <div className="all">
+          <div class="card">
+            <div class="imgBx">
+              <img
+                src="https://images.unsplash.com/photo-1532123675048-773bd75df1b4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+                alt="images"
+              />
+            </div>
+            <div class="details">
+              <h2>
+                SomeOne Famous
+                <br />
+                <span>Director</span>
+              </h2>
+            </div>
+          </div>
+          <div className="info">
+            <div className="color"></div>
+            <div className="links">
+              <a href="">
+                <AiFillFacebook className="space" />
+              </a>
+              <a href="">
+                <AiFillTwitterCircle className="space" />
+              </a>
+              <a href="">
+                <AiFillLinkedin className="space" />
+              </a>
+              <a href="">
+                <AiFillYoutube className="space" />
+              </a>
+            </div>
+          </div>
+        </div>
```

Comment:
Have you heard about components? Wouldn't it be easier to make this a separate component, and we pass their details via props whenever we want to add a new person?